apps/art-game/src/components/stencils/item.tsx

```diff
@@ -56,7 +68,7 @@ export const StencilItem = (props: any) => {
     >
       <Image
         loader={() => props.image}
-        src={props.image}
+        src={`https://moccasin-impossible-reptile-513.mypinata.cloud/ipfs/${ipfsHash}`}
```

Comment:
Please add the public endpoint provider with env with a default one please:

https://ipfs/io/